.github/ISSUE_TEMPLATE/upgrade_apps.md

```diff
@@ -76,6 +76,7 @@ https://github.com/giantswarm/loki-app
 * If won't upgrade, why?
 
 - [ ] tick this box if there is no new upstream release and no need for upgrade
+- [ ] Check Promtail
```

Comment:
@ubergesundheit , you mentioned this in planning today. Do you mean, like this? Or something else? Please check.

Reply:
```suggestion
- [ ] Check Promtail and release together with loki
```